packages/contracts-bedrock/test/dispute/WETH98.t.sol

```diff
@@ -4,32 +4,139 @@ import { Test } from "forge-std/Test.sol";
 import { WETH98 } from "src/dispute/weth/WETH98.sol";
 
 contract WETH98_Test is Test {
+    event Approval(address indexed src, address indexed guy, uint256 wad);
+    event Transfer(address indexed src, address indexed dst, uint256 wad);
+    event Deposit(address indexed dst, uint256 wad);
+    event Withdrawal(address indexed src, uint256 wad);
+
     WETH98 public weth;
     address alice;
+    address bob;
 
     function setUp() public {
         weth = new WETH98();
         alice = makeAddr("alice");
+        bob = makeAddr("bob");
         deal(alice, 1 ether);
     }
 
+    function test_getName_succeeds() public view {
+        assertEq(weth.name(), "Wrapped Ether");
+        assertEq(weth.symbol(), "WETH");
+        assertEq(weth.decimals(), 18);
+    }
+
     function test_receive_succeeds() public {
+        vm.expectEmit(true, true, true, true);
+        emit Deposit(alice, 1 ether);
         vm.prank(alice);
         (bool success,) = address(weth).call{ value: 1 ether }("");
         assertTrue(success);
         assertEq(weth.balanceOf(alice), 1 ether);
     }
 
     function test_fallback_succeeds() public {
+        vm.expectEmit(true, true, true, true);
+        emit Deposit(alice, 1 ether);
         vm.prank(alice);
         (bool success,) = address(weth).call{ value: 1 ether }(hex"1234");
         assertTrue(success);
         assertEq(weth.balanceOf(alice), 1 ether);
     }
 
-    function test_getName_succeeds() public view {
-        assertEq(weth.name(), "Wrapped Ether");
-        assertEq(weth.symbol(), "WETH");
-        assertEq(weth.decimals(), 18);
+    function test_deposit_succeeds() public {
+        vm.expectEmit(true, true, true, true);
+        emit Deposit(alice, 1 ether);
+        vm.prank(alice);
+        weth.deposit{ value: 1 ether }();
+        assertEq(weth.balanceOf(alice), 1 ether);
+    }
+
+    function test_withdraw_succeeds() public {
+        vm.prank(alice);
+        weth.deposit{ value: 1 ether }();
+        vm.expectEmit(true, true, true, true);
+        emit Withdrawal(alice, 1 ether);
+        vm.prank(alice);
+        weth.withdraw(1 ether);
+        assertEq(weth.balanceOf(alice), 0);
+    }
+
+    function test_withdraw_partialWithdrawal_succeeds() public {
+        vm.prank(alice);
+        weth.deposit{ value: 1 ether }();
+        vm.expectEmit(true, true, true, true);
```

Comment:
nit: should use `vm.expectEmit(address)` which implies true for each topic and also asserts on the address that emits the event